src/components/TodoItem.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styles from "./TodoItem.module.css";
+
+class TodoItem extends React.Component {
+  state = {
+    editing: false,
+  };
+  render() {
+    const completedStyle = {
+      fontStyle: "italic",
+      color: "#595959",
+      opacity: 0.4,
+      textDecoration: "line-through",
+    };
+    const { completed, id, title } = this.props.todo;
+
+    const handleEditing = () => {
+      handleEditing = () => {
+        this.setState({
+          editing: true,
+        });
+      }
+    }
+
+    let viewMode = {}
+    let editMode = {}
+
+    if (this.state.editing) {
+      viewMode.display = "none"
+    } else {
+      editMode.display = "none"
+    }
+
+    handleUpdatedDone = event => {
```

Comment:
- [ ] `HandleUpdatedDone` is used but undefined! Kindly make sure to declare it before using it in order to let your app render correctly